php/composer.json

```diff
@@ -2,12 +2,12 @@
     "name": "kata-setup/php",
     "description": "Initial code to start TDD katas",
     "require": {
-        "php": ">=7.4"
+        "php": ">=8.2.4"
```

Comment:
If we would require this latest version of PHP, what about providing a dockerfile to be able to setup this for those who doesn't have this concrete version installed in their local machines?